mongo/integration/unified/error_test.go

```diff
@@ -0,0 +1,160 @@
+// Copyright (C) MongoDB, Inc. 2017-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package unified
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+// ExpectedError represents an error that is expected to occur during a test. This type ignores the "isError" field in
+// test files because it is always true if it is specified, so the runner can simply assert that an error occurred.
+type ExpectedError struct {
+	IsClientError  *bool          `bson:"isClientError"`
+	ErrorSubstring *string        `bson:"errorContains"`
+	Code           *int32         `bson:"errorCode"`
+	CodeName       *string        `bson:"errorCodeName"`
+	IncluedLabels  []string       `bson:"errorLabelsContain"`
```

Comment:
```suggestion
	IncludeLabels  []string       `bson:"errorLabelsContain"`
```

Reply:
This was intentional. The field names in the JSON/YAML files are verb-based directives (e.g. `includeLabels`) but I chose to translate them into adjective-based struct fields (e.g. `IncludedLabels`). This was done for the `OmittedLabels` and `ExpectedResult` fields as well.

Reply:
Ah, I think it is still missing a `d` then. `IncludedLabels` not `IncluedLabels`.

Reply:
Sorry, misread the initial suggestion. Fixed.